src/lib/NumberFormatting.js

```diff
@@ -23,24 +23,43 @@ class NumberFormatting { // eslint-disable-line no-unused-vars
     /**
      * @param {number} value
      * @param {string} currency
+     * @param {number} [maxRelativeDeviation = .1]
      * @param {string} [locale = navigator.language]
      * @returns {string}
      */
-    static formatCurrency(value, currency, locale = navigator.language) {
+    static formatCurrency(value, currency, maxRelativeDeviation = 0.1, locale = navigator.language) {
```

Comment:
I don't understand how `maxRelativeDeviation` works. I think the default should be 0 (e.g. _no_ deviation) to get an exact number back, no? Why would I want a rounded number to two decimals (the starting `minimumFractionDigits`) when I not explicitly asked for it? And why is `minimumFractionDigits` limiting the decimals anyway? Shouldn't that be the job of `maximumFractionDigits`?

Reply:
@sisou
`minimumFractionDigits` and `maximumFractionDigits` both default to the decimal count typical for a currency. To get a higher precision, `minimumFractionDigits` has to be increased. `maximumFractionDigits` then defaults to `minimumFractionDigits` and does not have to be set manually.

@nibhar 
Not sure what you are suggesting in your last sentence but the method currently guarantees a max deviation, not a minimal decimal count, which is better I think and also what we discussed with overnice.